ingest/annotations.py

```diff
@@ -175,13 +183,13 @@ def validate_unique_header(self):
             for x in self.headers:
                 if x in seen_headers or seen_headers.add(x):
                     duplicate_headers.add(x)
-            msg = (
-                f"Duplicated metadata header names are not allowed: {duplicate_headers}"
-            )
+            msg = f"Duplicated header names are not allowed: {duplicate_headers}"
```

Comment:
```suggestion
            msg = f"Duplicate header names are not allowed: {duplicate_headers}"
```

Reply:
I agree "Duplicate" is more appropriate and more consistent. Thanks for the suggestion!